doc/reference/configuration/configuration_reference.rst

```diff
@@ -2861,11 +2861,12 @@ The ``replication`` section defines configuration parameters related to :ref:`re
 roles
 -----
 
-This section describes configuration parameters related to roles.
+This section describes configuration parameters related to application roles.
+An application role is a Lua module that implements specific functions or logic.
```

Comment:
While it is true that a role implements specific functions or logic, it doesn't differentiate the term 'role' from just 'module'.

If we say that a role is a managed code that can be enabled or disabled (started/stopped), it is closer to the idea of roles.

It is also important to say that a role is a Lua module and that role's name is a value passed to `require()` to obtain the module (unlike cartridge, where a role defines its own name in the code, which is different from module's name).